src/test/java/com/amazonaws/crypto/examples/DiscoveryDecryptionExampleTest.java

```diff
@@ -1,16 +0,0 @@
-// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
How are the example tested?

Reply:
Examples are now all run by https://github.com/acioc/aws-encryption-sdk-java/blob/keyring/src/test/java/com/amazonaws/crypto/examples/ExamplesTest.java#L64

This can be verified on Travis: https://travis-ci.org/github/aws/aws-encryption-sdk-java/jobs/733142171#L2358 (ExamplesTests is run with 33 tests - 1 test for each Example we have).